pulp_container/app/utils.py

```diff
@@ -90,8 +98,8 @@ def extract_data_from_signature(signature_raw, man_digest):
         return
 
     errors = []
-    for error in validator.iter_errors(sig_json):
-        errors.append(error.message)
+    for error in signature_validator.iter_errors(sig_json):
+        errors.append(f'{".".join(error.path)}: {error.message}')
```

Comment:
what's contained in error.path?

Reply:
The name of the JSON field which is considered invalid (e.g., the error message would look like this in case of an invalid schema version: "schemaVersion: 1 is less than 2").

Reply:
aha, gotcha, thanks!